### PR TITLE
CANS-1149. Update CANS Performance Test Scripts for CANS 2.0

### PR DESCRIPTION
**Pull request by denys-davydov**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-1149

## Description
1. Added steps to test Assessments Comparison feature
2. Added steps to test Assessment Change Log feature
3. Updated script to using /assessments/_initialize endpoint instead of /instument/{id} as a part of assessment initialization process (the change reflects the current cans approach to assessment initialization)
4. Removed usage of security/check_permission endpoint since it not used by the FE application any more

## Tests
- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] I ran all my tests locally which were green.
- [ ] My code adds no new issues to Code Climate
- [ ] I ran all the linters and static analyzers for the project (SonarQube, eslint, rubocop, etc).
- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.

### CODE REVIEW

File: jmeter/AssessmentsApi.jmx [hunk 1]
```diff
@@ -1329,21 +1331,85 @@ log.info(&quot;client_county_id = &quot; + vars.get(&quot;client_county_id&quot;
...
+                      <boolProp name="HTTPSampler.auto_redirects">false</boolProp>
+                      <boolProp name="HTTPSampler.use_keepalive">true</boolProp>
+                      <boolProp name="HTTPSampler.DO_MULTIPART_POST">false</boolProp>
+                      <stringProp name="HTTPSampler.embedded_url_re"></stringProp>
+                      <stringProp name="HTTPSampler.connect_timeout"></stringProp>
+                      <stringProp name="HTTPSampler.response_timeout"></stringProp>
+                    </HTTPSamplerProxy>
+                    <hashTree>
+                      <ResponseAssertion guiclass="AssertionGui" testclass="ResponseAssertion" testname="Response Assertion" enabled="true">
+                        <collectionProp name="Asserion.test_strings">
```
Comment (wmitchell-cambria) [inline jmeter/AssessmentsApi.jmx]: typo in `Assertion`?

**Comment by denys-davydov** [comment]
Retest this please

File: jmeter/AssessmentsApi.jmx [hunk 2]
```diff
@@ -1329,21 +1331,85 @@ log.info(&quot;client_county_id = &quot; + vars.get(&quot;client_county_id&quot;
...
```
Comment (wmitchell-cambria) [inline jmeter/AssessmentsApi.jmx]: Same here. (Were these generated by the GUI? If so, it's very strange.)

Comment (denys-davydov) [reply jmeter/AssessmentsApi.jmx]: Yes, it is generated by the GUI. It looks strange, but it works 😕

**Pull request merged**